sdk/python/feast/infra/registry_stores/sql.py

```diff
@@ -575,7 +584,7 @@ def get_user_metadata(
     def proto(self) -> RegistryProto:
         r = RegistryProto()
         project = ""
-        # TODO(achal): Support Infra object, and last_updated_timestamp.
+        # TODO(achal): Support last_updated_timestamp.
```

Comment:
is this comment still necessary? looks like you're supporting `last_updated_timestamp` below?

also should it be `last_updated_timestamp` instead of `last_update_timestamp` below?